### ISSUE TITLE
[BUG] Instagram++ miscellaneous settings

### ISSUE THREAD

**Issue by azryofficial** [eni9889/IG-PP-Issues#3]
There is a problem in miscellaneous settings, when I click "show follow status" it doesn't show, and when I click "unfollow without confirm" it's asking for confirmation.
NOTE: this problem is not only in my phone, it is in other phones too.

**Comment by eni9889** [comment]
The follow status should be fixed now. Also, please follow the instructions for posting bugs. I need to know: 

- [ ]  iOS Version
- [ ]  Instagram Version
- [ ]  IG ++ Tweak Version
- [ ]  Crash Report: http://tweakcrashed.com
- [ ]  Please label the issue either Jailbreak or Sideloaded

**Comment by azryofficial** [comment]
It still doesn't work, I uninstalled it and downloaded the app again. I guess a new update with a fix would work.
I can send a screenshot if you want

**Comment by JMccormick264** [comment]
Is this issue occurring on sideload or Jailbreak?

**Comment by azryofficial** [comment]
My phone isn't  jaikbroken. I downloaded the app in my another phone (ios 10.2.1) and in that too, the miscellaneous settings does not work.

**Comment by JMccormick264** [comment]
I just confirmed the following
 - Hide Bio Info in Profile View does not work (JB and Non JB)
 - Unfollow Without Confirm Is not working. Even with it off I am still asked if I want to unfollow (it May be the app itself) (JB and Non JB)
 - Full screen does work, you just need to restart the app
 - Save caption on ReGram does not work (JB and Non JB)
 - "Show follow status" has been fixed on JB. The 10.12 Instagram update broke that at first.
